pyinjective/core/broadcaster.py

```diff
@@ -62,6 +66,17 @@ def __init__(
         self._client = client
         self._composer = composer
         self._fee_calculator = fee_calculator
+        self._ofac_list = self.load_ofac_list()
+
+    @classmethod
+    def load_ofac_list(cls) -> List[str]:
+        try:
+            response = requests.get(OFAC_LIST_URL)
+            response.raise_for_status()
+            ofac_list = response.json()
+            return ofac_list
+        except requests.exceptions.RequestException as e:
+            raise OfacListFetchError(f"Error fetching OFAC list: {e}")
```

Comment:
Great helper method. But since the logic needs to execute a REST request, we should make this method an async method (and run the request using async logic)

Reply:
now this request is either run either in the init (which is obviously not async) of the objects, or in a sep call using something like `python3 pyinjective/ofac.py`

Reply:
Both contexts are not async.